db/models/models.js

```diff
@@ -5,4 +5,12 @@ exports.selectAllTopics = () => {
     return db.query('SELECT * FROM topics;')
     .then(({rows}) => {
         return rows;})
-}
\ No newline at end of file
+}
+
+exports.selectArticleById = (id) => {
+    return db
+        .query(`SELECT * FROM articles WHERE article_id = $1`, [id])
+        .then(({rows}) => {
+            return rows[0]
```

Comment:
We could check to see if the response is empty here, this would mean that the article id was not found. There's a hint for your missing test :)